### PR TITLE
Exclude Databricks Delta Lake test suites from CI

### PR DESCRIPTION
**Pull request by findinpath**
## Description

Exclude for the time being the Databricks suites test suites from the CI due to incorrect secrets which is causing the tests to fail.

See https://github.com/trinodb/trino/actions/runs/4370055105/jobs/7646013312 for details.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
@nineinchnick do i understand correctly we won't be able to `/test-with-secrets` a future revert?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -831,23 +831,19 @@ jobs:
...
               config: hdp3
             - suite: suite-delta-lake-databricks73
-              ignore exclusion if: >-
-                ${{ secrets.DATABRICKS_TOKEN != '' }}
+              config: default
             - suite: suite-delta-lake-databricks91
               config: hdp3
             - suite: suite-delta-lake-databricks91
-              ignore exclusion if: >-
-                ${{ secrets.DATABRICKS_TOKEN != '' }}
```
Comment (findepi) [inline .github/workflows/ci.yml:840]: @findinpath @alexjo2144 @nineinchnick 
or should I just delete DATABRICKS_TOKEN secret for now?

Comment (findinpath) [reply .github/workflows/ci.yml:840]: I would greatly be in favor of removing `DATABRICKS_TOKEN` secret.

Comment (findepi) [reply .github/workflows/ci.yml:840]: DATABRICKS_TOKEN removed.

**Pull request closed without merging**